examples/web-app/src/components/Home/Home.js

```diff
@@ -0,0 +1,196 @@
+import React, { useState, useEffect } from "react";
+import { UserFactory } from "nightfall-sdk";
+import "./Home.css";
+
+export default function User() {
+  const [userAddress, setUserAddress] = useState();
+  const [userNightfallAddress, setUserNightfallAddress] = useState();
+  const [userRecipient, setUserRecipient] = useState();
```

Comment:
These two can be deleted, we never use them (`userRecipient` and `setUserRecipient`).